docker-compose.yml

```diff
@@ -19,9 +18,7 @@ services:
       - client_id=
       - client_secret=
       - state_password=
-    # working_dir: /app
-    # entrypoint: ["/wait_for_postgres.sh", "db"]
-    command: tail -f /dev/null
+#   command: ["tail", "-f", "/dev/null"]
```

Comment:
I think we can safely remove this line. If it's needed it is possible to bring it back later.